AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -749,13 +759,13 @@ - (NSMutableDictionary *)getReportingParametersForUpdatedRelease:(NSString *)upd
 
   // Check if we need to report release installation.
   NSString *lastDownloadedReleaseHash = [MS_USER_DEFAULTS objectForKey:kMSDownloadedReleaseHashKey];
-  if (!lastDownloadedReleaseHash) {
+  if (lastDownloadedReleaseHash == nil) {
     MSLogDebug([MSDistribute logTag], @"Current release was already reported, skip reporting.");
     return nil;
   }
 
   // Skip if downloaded release not installed yet.
-  if (![lastDownloadedReleaseHash isEqualToString:currentInstalledReleaseHash]) {
+  if (![lastDownloadedReleaseHash containsString:currentInstalledReleaseHash]) {
```

Comment:
Not a big deal but any reasons for the change?

Reply:
The reason is that `lastDownloadedReleaseHash` can be a comma separated string of several hashes. Will rename the variable to reflect it.